morpho-labs/zapper-studio
# feat(morpho-blue): first iteration

## Description



## Checklist

- [x] I have followed the [Contributing Guidelines](https://github.com/Zapper-fi/studio/blob/main/CONTRIBUTING.md)
- [x] (optional) As a contributor, my Twitter handle is: @TomReppelin

## How to test?


- Morpho Blue adapter. 

Test it by running 

`pnpm build && pnpm start`

and go there:

http://localhost:5001/apps/morpho/positions?network=ethereum&groupIds%5B%5D=morpho-blue

One should see something like this:
<img width="789" alt="image" src="https://github.com/morpho-labs/zapper-studio/assets/75627352/570ed12e-4d81-4c1b-8cee-5ffc1b5526d7">
